connexion/apis/flask_api.py

```diff
@@ -154,6 +154,12 @@ def _build_flask_response(cls, mimetype=None, content_type=None,
         flask_response = flask.current_app.response_class(**kwargs)  # type: flask.Response
 
         if status_code is not None:
+            try:
```

Comment:
Please use ``hasattr`` here since the most common use case is ``status_code`` being an integer.

[``hasattr`` is faster than ``try... except AtributeError`` for cases where the attribute doesn't exist](https://stackoverflow.com/a/903238).

Reply:
Yes, and you get to repeat yourself while doing the check. It'd be even faster if just didn't support enums at all!

Will update.